assets/l10n/intl_en.arb

```diff
@@ -2969,7 +2969,7 @@
   "groupName": "Group name",
   "descriptionHelper": "You can provide an optional description for your group.",
   "groupNameCannotBeEmpty": "Group name cannot be empty",
-  "sharedMediaAndFiles": "Shared media and files",
+  "sharedMediaAndLinks": "Shared media and links",
```

Comment:
if only change it, other language still not update too.
Solution: delete the old one for all `intl_**` file